src/InsertionsClient.Console/ConsoleApp/InputLoading.cs

```diff
@@ -111,5 +113,185 @@ internal static List<string> LoadManifestPaths(string manifestPaths, out int inv
 
             return results;
         }
+    
+        /// <summary>
+        /// Parses the input string into a filter that will decide if a build should be inserted or not.
+        /// </summary>
+        /// <param name="filterString">Input string to be parsed.</param>
+        /// <param name="buildFilter">Filter generated from the input string.</param>
+        /// <returns>True if loading succeeded. False otherwise. </returns>
+        internal static bool LoadBuildFilter(string filterString, out Predicate<Build>? buildFilter)
+        {
+            buildFilter = null;
+
```

Comment:
please consider modularizing this method...can a couple of the distinct things it does become methods?

Reply:
+1 vote for modularizing